Marlin/language_en.h

```diff
@@ -561,22 +561,35 @@
     #define MSG_INFO_PRINT_COUNT              "Print Count"
   #endif
   #ifndef MSG_INFO_COMPLETED_PRINTS
-    #define MSG_INFO_COMPLETED_PRINTS         "Completed  "
+    #define MSG_INFO_COMPLETED_PRINTS         "Completed"
   #endif
   #ifndef MSG_INFO_PRINT_TIME
-    #define MSG_INFO_PRINT_TIME               "Total Time "
+    #define MSG_INFO_PRINT_TIME               "Total print time"
+  #endif
+  #ifndef MSG_INFO_PRINT_LONGEST
+    #define MSG_INFO_PRINT_LONGEST            "Longest job time"
+  #endif
+  #ifndef MSG_INFO_PRINT_FILAMENT
+    #define MSG_INFO_PRINT_FILAMENT           "Extruded total"
```

Comment:
Yeah, it was my idea. Looks weird on graphical LCD, but on 16x2 you can't do much about it. This fits perfectly in 16 chars with 5-digit year number, which is the limit for `int32`.